### ISSUE TITLE
[Windows 2008 ] WriteProcessMemory: '(originated from ReadProcessMemory(ProcessParameters))'. (Python 2.7)

### ISSUE THREAD

**Issue by hejink** [giampaolo/psutil#1549]
**Platform**
* { OS version }     (also add appropriate OS issue label (linux, windows, ...))
* { psutil version }  (use "pip show psutil")

**Bug description**
{ a clear and concise description of what the bug is }

```
traceback message (if any)
```

```python
code to reproduce the problem (if any)
```

**Test results**
```
output of `python -c psutil.tests` (failures only, not full result)
```
{ you may want to do this in order to discover other issues affecting your platform }
{ if failures look unrelated with the issue at hand open another ticket }

**Comment by hejink** [comment]
python27

**Comment by hejink** [comment]
occasionally

**Comment by giampaolo** [comment]
I should have fixed this already. What psutli version are you using?

**Comment by hejink** [comment]
occasionally

> I should have fixed this already. What psutli version are you using?

5.6.2

**Comment by giampaolo** [comment]
It was fixed in 5.6.3.

**Comment by hejink** [comment]
> It was fixed in 5.6.3.

OK, thank you, I will upgrade
